config/foreman.migrations/160614145332_foreman_expire_hosts.rb

```diff
@@ -0,0 +1,2 @@
+answers['foreman::plugin::expire_hosts'] ||= false
```

Comment:
@domcleal: Sorry for this, i just copied the command from the kafo docs. Pushed a new commit.

Is `date +%y%m%d%H%M%S` the correct command on your machine?
I need `date +%Y%m%d%H%M%S` both on OS X and RHEL...